### PR TITLE
OnBoard page, BE, FE, routes

### PR DESCRIPTION
**Pull request by svitaly**
Work is still in progress

### CODE REVIEW

File: client/src/services/languages.js [hunk 1]
```diff
@@ -21,29 +21,23 @@ const getLanguages = async (dispatch) => {
...
+      },
+    };
+
+    const response = await request.putToAPI(config);
+    dispatch({
+      type: "UPDATE_USER_LANGUAGES_SUCCESS",
+      user: response.data,
+    });
+  } catch (e) {
+    dispatch({ type: "UPDATE_USER_LANGUAGES_ERROR", error: { ...e } });
```
Comment (Affeeq) [inline client/src/services/languages.js:39]: A lot of the updated services file do not have that initial dispatch to change loading state. I think that should be added to any service that is making an API call.

File: server/routes/index.js [hunk 2]
```diff
@@ -43,6 +43,7 @@ router.post("/register", async (req, res) => {
     return res.status(201).send({
       message: "User created!",
       token,
```
Comment (Affeeq) [inline server/routes/index.js:45]: you can remove this token

File: server/routes/index.js [hunk 3]
```diff
@@ -67,13 +68,13 @@ router.post("/login", async (req, res) => {
     return res.status(200).send({
       message: "User logged in!",
       token,
```
Comment (Affeeq) [inline server/routes/index.js:70]: right here as well

File: client/src/services/languages.js (hunk 1 above)
Comment (svitaly) [reply client/src/services/languages.js:39]: In Login and Signup there is no need to change state to 'loading: true' , bc if we log in or signup succesfully loading will be changed to true with getUser service and before that we can return errors from server in realtime

File: client/src/reducers/index.js [hunk 4]
```diff
@@ -9,6 +9,21 @@ export default (state, action) => {
...
+    }
+    case "LOGOUT_USER_SUCCESS": {
+      return { ...state, user: {}, loading: false, error: null };
+    }
+    case "LOGIN": {
+      return { ...state, user: action.user, loading: false, error: null };
+    }
+    case "SIGNUP": {
+      return { ...state, user: action.user, loading: false, error: null };
+    }
```
Comment (Affeeq) [inline client/src/reducers/index.js:26]: I think there is a similar reducer for logout in there, it is called REMOVE_USER etc. Can remove that if using these reducers.

File: client/src/services/languages.js (hunk 1 above)
Comment (Affeeq) [reply client/src/services/languages.js:39]: got it!

File: client/src/reducers/index.js (hunk 4 above)
Comment (svitaly) [reply client/src/reducers/index.js:26]: Decided to leave it for now, maybe we will use it the future, if not - will remove it

File: client/src/services/languages.js (hunk 1 above)
Comment (svitaly) [reply client/src/services/languages.js:39]: Added initial dispatch in language.js service

**Pull request merged**